cpp/examples/ide_secir_ageres.cpp

```diff
@@ -0,0 +1,136 @@
+    vec_init[(int)mio::isecir::InfectionTransition::InfectedSymptomsToRecovered]          = 4.0;
+    vec_init[(int)mio::isecir::InfectionTransition::InfectedSevereToInfectedCritical]     = 1.0;
+    vec_init[(int)mio::isecir::InfectionTransition::InfectedSevereToRecovered]            = 1.0;
+    vec_init[(int)mio::isecir::InfectionTransition::InfectedCriticalToDead]               = 1.0;
+    vec_init[(int)mio::isecir::InfectionTransition::InfectedCriticalToRecovered]          = 1.0;
+    //Second AgeGroup.
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::SusceptibleToExposed]                 = 25.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::ExposedToInfectedNoSymptoms]          = 15.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::InfectedNoSymptomsToInfectedSymptoms] = 8.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::InfectedNoSymptomsToRecovered]        = 4.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::InfectedSymptomsToInfectedSevere]     = 1.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::InfectedSymptomsToRecovered]          = 4.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::InfectedSevereToInfectedCritical]     = 1.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::InfectedSevereToRecovered]            = 1.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::InfectedCriticalToDead]               = 1.0;
+    vec_init[num_transitions + (int)mio::isecir::InfectionTransition::InfectedCriticalToRecovered]          = 1.0;
+    vec_init = vec_init * dt;
+    // Add initial time point to time series.
+    init.add_time_point(-10, vec_init);
+    // Add further time points until time 0.
+    while (init.get_last_time() < -dt / 2) {
+        init.add_time_point(init.get_last_time() + dt, vec_init);
+    }
+
+    // Initialize model.
+    mio::isecir::Model model(std::move(init), N, deaths, num_agegroups, std::vector<ScalarType>(num_agegroups, 100.));
+
+    // Uncomment these lines to use a different method to initialize the model using the TimeSeries init.
+    // Initialization method with Susceptibles.
+    //model.m_populations.get_last_value()[(Eigen::Index)mio::isecir::InfectionState::Susceptible] = 1000;
+    // model.m_populations.get_last_value()[(Eigen::Index)mio::isecir::InfectionState::Count +
+    //                                      (Eigen::Index)mio::isecir::InfectionState::Susceptible] = 1000;
+    // Initialization mehtod with Recovered.
+    //model.m_populations.get_last_value()[(Eigen::Index)mio::isecir::InfectionState::Recovered] = 0;
+    // model.m_populations.get_last_value()[(Eigen::Index)mio::isecir::InfectionState::Count +
+    //                                      (Eigen::Index)mio::isecir::InfectionState::Recovered] = 0;
+
```

Comment:
Is this left over from development?

Reply:
No, if you uncomment these you can use another initialization method.